controllers/session_test.go

```diff
@@ -252,6 +252,50 @@ var _ = Describe("SessionController", func() {
 		})
 	})
 
+	Describe("GET /signout", func() {
+		Context("given user is already signed in", func() {
+			It("redirects to sign in path", func() {
+				user := FabricateUser("dev@nimblehq.co", "password")
+				body := GenerateRequestBody(map[string]string{})
```

Comment:
How about like this
```suggestion
				body := GenerateRequestBody(nil)
```

Is it more clearer to send `nil` to the function?